collections/_article/uid-changes-coming-to-godot-4-4.md

```diff
@@ -71,17 +73,29 @@ While Godot must import many resource types before they can be used in a project
 
 The main examples of resources that don't need to be imported are scripts and shaders, hence the dedicated `.uid` files.
 
+It has been suggested that we could still use `.meta` files for scripts and shaders, and use them to store more information than just UIDs, such as user-defined metadata. For the time being we prefer to focus on the problem at hand (the need of UIDs for better refactoring possibility) and not future proof things. If we ever change our mind, it would be easy enough to migrate information from `.uid` files to whichever new container we decide to use.
```

Comment:
```suggestion
It has been suggested that we could still use `.meta` files for scripts and shaders, and use them to store more information than just UIDs, such as user-defined metadata. For the time being we prefer to focus on the problem at hand (the need of UIDs for better refactoring possibility) and not future-proof things. If we ever change our mind, it would be easy enough to migrate information from `.uid` files to whichever new container we decide to use.
```